cl/disclosures/utils.py

```diff
@@ -42,3 +42,29 @@ def has_been_extracted(data: Dict[str, Union[str, int, list]]) -> bool:
     return FinancialDisclosure.objects.filter(
         download_filepath=url, has_been_extracted=True
     ).exists()
+
+
+def make_disclosure_data(person) -> Tuple:
+    """
+
+    :param person:
+    :return:
+    """
+    forms = (
+        person.financial_disclosures.all()
+        .order_by("-year")
+        .values_list("year", "id")
+    )
+    years = []
+    ids = []
+    for yr, id in forms:
+        years.append((str(yr)))
+        ids.append(str(id))
+    for x in set(years):
+        number = 0
+        for i in range(0, len(years)):
+            if years[i] == x:
+                number += 1
+                if number >= 2:
+                    years[i] += f" ({number})"
+    return ",".join(years), ",".join(ids)
```

Comment:
This feels a bit awkward to have two objects here. Why not have a dict:

```
{
  "2000": 12,
  "2000 (2)": 13,
}
```

Reply:
this was a stack pull for lack of mental energy at the time.  I can refactor this im sure.

Reply:
okay.  actually  I don't see a huge benefit from using a dictionary here.  I like the return tuple but happy to be convinced otherwise.

Reply:
There isn't a huge benefit, since I realize you have to split it up for your data attribtues later, but it seemed like you have data mapping from one thing to another, which screams for a dict. No biggie.